wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -80,9 +80,9 @@ Probationary Beatmap Nominators are expected to meet different activity requirem
 Please see the [beatmap veto](/wiki/People/Beatmap_Nominators/Beatmap_Veto) page for full information on the veto process.
 
 - **Do not veto beatmaps of game modes you are not formally qualified for.** You are not allowed to nominate them so blocking a nomination without the ability to renominate the beatmap makes no sense. This does not apply to vetoes regarding audio, backgrounds, or other content that does not directly pertain to the beatmap's game mode.
-- **You can veto a nominated/qualified beatmap by posting the issues you think need to be addressed before reaching Ranked status in the form of problems in the respective modding discussion.** This will reset all nominations the beatmap has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must include a clear problem and reasoning for stopping the ranking process of the beatmap. You must participate in the resulting discussion and attempt to reach a mutual agreement.
+- **You can veto a beatmap by posting the issues you think need to be addressed before reaching Ranked status in the form of Problem stamps in the respective modding discussion.** This will reset any nominations the beatmap has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must include a clear problem and reasoning for stopping the ranking process of the beatmap. You must participate in the resulting discussion and attempt to reach a mutual agreement.
```

Comment:
```suggestion
- **You can veto a beatmap by posting the issues you think need to be addressed before reaching Ranked status in the form of problem stamps in the respective modding discussion.** This will reset any nominations the beatmap has received and thereby stop it from proceeding to Ranked. Other nominators will be unable to nominate it again until the veto is lifted. The veto post must include a clear problem and reasoning for stopping the ranking process of the beatmap. You must participate in the resulting discussion and attempt to reach a mutual agreement.
```